docs/developers/celestia-node.md

```diff
@@ -0,0 +1,28 @@
+# Celestia Node
+
+This tutorial goes over installing and building Celestia Node. This
+tutorial presumes you completed the steps in setting up your own
+environment [here](../developers/environment).
+
+## Install Celestia Node
+
+Install the Celestia Node binary, which will be used to run the Bridge Node.
```

Comment:
Is there a specific reasons this mentions a bridge node here? the instructions below are independent of the mode the node is ran in later 🤔

Reply:
You're right, I think it crept up from copy-pasta of other page. Will remove it. Good catch!